services/blockchain-indexer/shared/dataService/business/schemas.js

```diff
@@ -13,24 +13,46 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+const { getRegisteredModules, MODULE } = require('../../constants');
 const { requestConnector } = require('../../utils/request');
 
 let allSchemas;
 
+const getAuthMultiSigRegMsgSchema = async () => {
```

Comment:
I think its better to move this method `getAuthMultiSigRegMsgSchema` to `services/blockchain-indexer/shared/dataService/business/auth` as it is relevant to auth module.

Reply:
Applied.